identity/app/model/EmailSubscriptions.scala

```diff
@@ -220,19 +228,21 @@ object EmailSubscriptions {
       "Green light",
       "news",
       "Environment",
-      "In each weekly edition our editors highlight the most important stories of the week including data, opinion pieces and background guides. We'll also flag up our best video, picture galleries, podcasts, blogs and green living guides.",
-      "",
+      "In each weekly edition our editors highlight the most important environment stories of the week including data, opinion pieces and background guides. We'll also flag up our best video, picture galleries, podcasts, blogs and green living guides.",
+      "Every Friday",
       "38",
-      subscribedTo = subscribedListIds.exists{ x => x == "38" }
+      subscribedTo = subscribedListIds.exists{ x => x == "38" },
+      exampleUrl = Some("http://www.theguardian.com/environment/series/green-light/latest/email")
```

Comment:
Should we use [guardian.page.host](https://github.com/guardian/frontend/blob/5e03ef5dbbd7d1a49f598c3e2b3761690b505b37/common/conf/env/PROD.properties#L52) for these?